.github/workflows/test.yaml

```diff
@@ -17,13 +18,20 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
     steps:
-    - uses: actions/checkout@v1     
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Upgrade pip
+      run: python -m pip install --upgrade pip
     - uses: actions/cache@v2
       with:
         path: ~/.cache/pip
-        key: ${{ runner.os }}-pip-${{ hashFiles('requirements_for_test.txt') }}
+        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
         restore-keys: |
           ${{ runner.os }}-pip-
-    - run: /bin/bash -c "pip3 install -r requirements_for_test.txt --no-cache-dir && make test"
+    - name: Run tests
+      run: /bin/bash -c "pip install -r requirements_for_test.txt && make test"
       env:
-        SQLALCHEMY_DATABASE_URI: postgresql://postgres:postgres@postgres:5432/test_notification_api
+        SQLALCHEMY_DATABASE_URI: postgresql://postgres:postgres@localhost:5432/test_notification_api
```

Comment:
Tests were run inside a container before, now directly on the VM. This is a required change
https://github.com/actions/example-services/blob/7ae0b005cc273e2000e12ccf54f01e796c22b3e4/.github/workflows/postgres-service.yml#L39-L40